modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/card-shortcut/CardShortcut.es.js

```diff
@@ -20,6 +20,11 @@ export default ({fields}) => {
 	const [itemSelected, setItemSelected] = useState(null);
 	const {portletNamespace} = useContext(SidebarContext);
 
+	if (itemSelected == null) {
+		if (window.location.hash) {
+			setItemSelected(window.location.hash.match(/.*(\d+)$/)[1]);
```

Comment:
@alinedoleron, can we extract this regex to a constant in order to improve code legibility? My suggestion is something like [this](https://github.com/liferay/liferay-portal/blob/4ac7f1ae7e2bd3c9e776a3dff657ee37feaa95de/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/util/strings.es.js#L19-L21). When we keep a regex in the code like this, isn't very straightforward for another developer understand it once he/she isn't aware of the context of this story, does it make sense to you?